testunit.go

```diff
@@ -154,6 +139,49 @@ func prepareBundle(validateObj string) {
 	Mutex.Unlock()
 }
 
+func (unit *TestUnit) GenerateConfigs() {
+	args := splitArgs(unit.Args)
+
+	logrus.Debugf("Args to the ocitools generate: ")
+	for _, a := range args {
+		logrus.Debugln(a)
+	}
+	Mutex.Lock()
+	_, err := utils.ExecGenCmd(args)
+	if err != nil {
+		logrus.Fatalf("Generate *.json err: %v\n", err)
+	}
+
+	copy("./plugins/runtime.json-"+unit.Name, "./plugins/runtime.json")
+	if err != nil {
+		logrus.Fatalf("copy to runtime.json-%v, %v", unit.Name, err)
+	}
+
+	copy("./plugins/config.json-"+unit.Name, "./plugins/config.json")
+	if err != nil {
+		logrus.Fatalf("copy to config.json-%v, %v", unit.Name, err)
+	}
+	Mutex.Unlock()
+}
+
+func splitArgs(args string) []string {
+
+	argsnew := strings.TrimSpace(args)
+
+	argArray := strings.Split(argsnew, "--")
+
+	lenth := len(argArray)
+	resArray := make([]string, lenth-1)
+	for i, arg := range argArray {
+		if i == 0 || i == lenth {
+			continue
+		} else {
+			resArray[i-1] = "--" + strings.TrimSpace(arg)
+		}
+	}
+	return resArray
+}
+
 func copy(dst string, src string) error {
 	in, err := os.Open(src)
```

Comment:
@zenlinTechnofreak May be we can consider put  `copy` function into `utils/file.go`